mlir/test/Dialect/Tosa/canonicalize.mlir

```diff
@@ -622,3 +622,24 @@ func.func @fold_abs_abs(%arg0: tensor<?x1xf32>) -> tensor<?x1xf32> {
   %1 = "tosa.abs"(%0) : (tensor<?x1xf32>) -> tensor<?x1xf32>
   return %1 : tensor<?x1xf32>
 }
+
```

Comment:
Can you try to align to location of the tests with that of the canonicalizer? As discussed recently, it would be nicer if we don't always append and get conflicts. Seems like the files partially follow this, but our additions screw this oftentimes 😅